build.py

```diff
@@ -1319,7 +1319,10 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu, target_mach
         )
         trtllm_buildscript = importlib.util.module_from_spec(spec)
         exec(response.content, trtllm_buildscript.__dict__)
-        df += trtllm_buildscript.create_postbuild()
+        # FIXME: Specify these versions via build flag?
```

Comment:
I think better alternative is to put it in Triton version map at the top.

Reply:
Great idea. Updated the file.